MesnardBarba-CiSE.md

```diff
@@ -118,26 +118,27 @@ Still, we ran against an obscure trick of the trade that changed our results com
+(word choice) an Eulerian mesh for the fluid, and a Lagrangian mesh for the solid. 
 The forces exerted by the fluid on the body, and vice versa, appear as an additional integral equation and interpolation schemes between the two meshes. 
-The role of these is to make the fluid "stick" to the wall (no-slip boundary condition) and allow the body to feel aerodynamic forces (lift and drag).
+(The forces don't appear as interpolation schemes, rather the interpolation schemes are used to compute the forces)
+The role of these (unclear antecedent - these == "forces" or these == "additional integral eq and interpolation scheme") is to make the fluid "stick" to the wall (no-slip boundary condition) and allow the body to feel aerodynamic forces (lift and drag).
 Our cuIBM code uses a variant called the immersed boundary projection method,^(4) while IBAMR uses a form called the "direct-forcing" method. 
-Despite the variations, the essence is the same, and it is reasonable to assume they would work similarly.
+Despite the variations, the two are essentially the same, and it is reasonable to assume they would work similarly.
 
 We already know that boundary conditions at the outlet of the computational domain can be problematic. This is no different with immersed boundary methods. 
 Our first attempt with IBAMR used a zero-gradient velocity boundary condition at the outlet. 
-This resulted in some blockage effect when the wake vortices reach the domain boundary: strong vorticity rebounds from the artificial boundary and propagates back to the domain (Figure 5). 
-Of course, this is unphysical and the result unacceptable. 
-After a long search in the literature and in the code documentation, we discovered that IBAMR needs us to select a "stabilized outlet," which is a boundary condition that acts like a force pushing the vortices out. 
+This results in a(?) blockage effect when the wake vortices reach the domain boundary: strong vorticity rebounds from the artificial boundary and propagates back to the domain (Figure 5). 
+Of course, this is unphysical and the result is unacceptable. 
+After a long search in the literature and in the code documentation, we discovered that IBAMR requires the selection of a "stabilized outlet," which is a boundary condition that acts like a force pushing the vortices out of the domain. 
 (IBAMR does not provide a convective/advective boundary condition.) 
 With this new configuration, the simulations of the snake profile resulted in a wake that looked physical, but a computed lift coefficient that was considerably different from our published study (Figure 6). 
-Another deep dive in the literature led us to notice that a benchmark example described in a paper describing extensions to IBAMR^(5) was set up in an unexpected way: 
+Another deep dive in the literature led us to notice that a benchmark example defined in a paper describing extensions to IBAMR^(5) was set up in an unexpected way: 
```

Comment:
I don't understand this suggestion. The example was indeed _described_ in the paper. I don't see how _defined_ applies here.